### PR TITLE
Clean up sorting and add sort functions

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: projects/epsilon-blueprint/src/lib/table/table.component.html [hunk 1]
```diff
@@ -34,15 +34,15 @@
...
             <th
               *ngIf="column.isSortable; else columnNotSortable"
               class="th-sortable"
               scope="col"
               [hidden]="!column.isColumnDisplayed"
-              [attr.aria-sort]="isSortActive ? getAriaSortOrder(i) : 'null'"
+              [attr.aria-sort]="getAriaSortOrder(column.key)"
             >
-              <button (click)="applySort(column.key, i)">
+              <button (click)="applySort(column.key)">
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table.component.html:45]: Should this fn just be named 'sort'?

File: projects/epsilon-blueprint/src/lib/table/table.component.ts [hunk 2]
```diff
@@ -50,17 +50,18 @@ export class TableComponent implements OnInit, OnChanges {
       'Actions Menu': 'Actions Menu',
       'Column Selector': 'Column Selector:',
       'Default': '(Default)',
-      'Showing numVisible out of numTotal': 'Showing #{numVisible} out of #{numTotal}',
+      'Showing numVisible out of numTotal':
+        'Showing #{numVisible} out of #{numTotal}',
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table.component.ts:54]: Do we want this line break? Assuming it was a Prettier thing

File: projects/epsilon-blueprint/src/lib/table/table.component.ts [hunk 3]
```diff
@@ -50,17 +50,18 @@ export class TableComponent implements OnInit, OnChanges {
...
+      'numResults results': '#{numResults} result(s)',
       'Number of Rows': 'Number of Rows per Page',
       'First Page': 'First Page',
       'Previous Page': 'Previous Page',
       'Next Page': 'Next Page',
       'Last Page': 'Last Page',
       'Display Density Options': {
         'Comfortable': 'Comfortable',
-        'Compact': 'Compact'
+        'Compact': 'Compact',
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table.component.ts:64]: Can we delete the trailing comma?

File: projects/epsilon-blueprint/src/lib/table/table.component.ts [hunk 4]
```diff
@@ -213,59 +243,61 @@ export class TableComponent implements OnInit, OnChanges {
...
-        }
-      );
+  getAriaSortOrder(key: string): string {
+    // Set aria attributes before data is loaded, but after column headers have rendered
+    if (this.isDataLoading && key === this.sortColumnKey) {
+      return this.properties.sort.defaultSortOrder;
     }
-    if (this.columnIndex === index) {
-      this.isSortActive = true;
+    // Set aria attributes after data is loaded
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table.component.ts:280]: This repeats 1/2 of the comment on 276. Do we need both?

File: projects/epsilon-blueprint/src/lib/table/table.component.ts (hunk 2 above)
Comment (APJohns) [reply projects/epsilon-blueprint/src/lib/table/table.component.ts:54]: It did this to keep to the character limit. We should keep it if we want to stick to our rules.

File: projects/epsilon-blueprint/src/lib/table/table.component.ts (hunk 4 above)
Comment (APJohns) [reply projects/epsilon-blueprint/src/lib/table/table.component.ts:280]: I think so, the first denotes before, this denotes after.

**Pull request merged**